framework/lib/components/createable-select-dropdown/types.ts

```diff
@@ -0,0 +1,49 @@
+import type { Option } from '@northlight/ui'
+
+export interface CreatableSelectDropdownProps {
+  /**
+   * An array of "Option" objects that represents the initial options available in the dropdown.
+   * Each "Option" object must have a "label" and a "value" property (both strings).
+   */
+  standardOptions: Option[]
+
+  /**
+   * A callback function that is called whenever the selected option changes.
+   * This function receives the newly selected "Option" object as its only argument.
+   */
+  onOptionChange: (option: Option) => void
+
+  /**
+   * Optional placeholder text displayed when no option is selected and the dropdown is not focused.
+   * Defaults to 'Select or create...' if not provided.
+   */
+  initialPlaceholder?: string
+
+  /**
+   * Optional placeholder text displayed when the dropdown is
+   * focused and the user is prompted to create a new option.
+   * Defaults to 'Enter text...' if not provided.
+   */
+  addOptionPlaceholder?: string
+
+  /**
+   * Optional "Option" object representing the option to create a new item within the dropdown.
+   * This option is typically displayed at the top of the dropdown list.
+   * Defaults to "{ label: 'Add option...', value: 'custom-option' }" if not provided.
+   */
+  creationOption?: Option
+
+  /**
+   * Optional CSS width property value for the dropdown container.
+   * Specify the width of the dropdown. Defaults to '100%' if not provided.
+   */
+  width?: string
+
+  /**
+   * An optional string that determines the overall styling of the select field.
+   * Accepted values are 'outline', 'filled', 'flushed', or 'unstyled'.
+   * The default value is 'outline'.
+   *
+  */
+  variant?: string
```

Comment:
`TagVariant` instead of `string`

Reply:
I think you mean Variant, added it now, from node:modules:
export type Variant = "outline" | "filled" | "flushed" | "unstyled" | (string & {});